targets/esp8266/Makefile.esp8266

```diff
@@ -46,61 +38,35 @@ ifneq ($(MFORCE32),)
 	ESP_CFLAGS += -mforce-l32
 endif
 
-# include path
-ESP_LIBS_INC :=$(CURDIR)/targets/esp8266/include
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)/espressif
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)/freertos
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)/json
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)/lwip
-ESP_LIBS_INC :=$(ESP_LIBS_INC);$(ESP_INC2)/ssl
-
-# jerry related source files
-JERRY_BUILD_FILES := $(SRCPATH)/jerry_extapi.c
-JERRY_BUILD_FILES := $(JERRY_BUILD_FILES);$(SRCPATH)/jerry_run.c
-
-
 .PHONY: jerry js2c mkbin check-env flash clean
 
-
 all: check-env jerry js2c mkbin
 
-
 jerry:
-	mkdir -p $(INTERM)
-	mkdir -p $(OUTPUT)
+	mkdir -p $(BUILD_DIR)
 	mkdir -p $(COPYTARGET)
-	cmake -B$(INTERM) -H./ \
+	cmake -B$(BUILD_DIR) -H./ \
 	 -DENABLE_LTO=OFF \
-	 -DENABLE_VALGRIND=OFF \
-	 -DCMAKE_TOOLCHAIN_FILE=build/configs/toolchain_external.cmake \
+	 -DFEATURE_VALGRIND=OFF \
+	 -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain_external.cmake \
 	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=xtensia-lx106 \
 	 -DEXTERNAL_CMAKE_C_COMPILER=xtensa-lx106-elf-gcc \
 	 -DEXTERNAL_CMAKE_C_COMPILER_ID=GNU \
-	 -DEXTERNAL_BUILD_ENTRY_FILE="$(JERRY_BUILD_FILES)" \
+	 -DJERRY_LIBC=OFF \
```

Comment:
As jerry-libm is not needed, it should be disabled for cmake, too (`-DJERRY_LIBM=OFF`)

Reply:
This isn't fixed. Jerry libm is till on. Is jerry-libm needed or not?

Reply:
Conclusion was that jerry-libm is still needed as toolchain's own libm is not featureful enough. Might be that this was discussed in another comment thread.